analysis/src/python/data_collection/hyperskill/api/users.py

```diff
@@ -0,0 +1,64 @@
+import datetime
+from dataclasses import dataclass
+from typing import List, Optional
+
+from analysis.src.python.data_collection.api.platform_objects import BaseRequestParams, Object, ObjectResponse
```

Comment:
Could you please in each file in the API package add some description on the top of the file of what the file contains? I mean we have a lot of models and it is difficult to understand clearly all entities for new users.